File: MyAPI.NET5_Angular/Front/ProEventos-APP/src/app/models/Evento.ts
import { Lote } from "./Lote";
import { Palestrante } from "./Palestrante";
import { RedeSocial } from "./RedeSocial";

export interface Evento {
    eventoId: number ;
    local: string ;
    dataEvento? : Date ;
    tema: string ;
    qtdPessoas: number ;
    imagemURL: string ;
    telefone: string ;
    email: string ;
    lotes:  Lote[] ;
    redeSocial:  RedeSocial[];
    palestrantesEventos: Palestrante[];
}

File: MyAPI.NET5_Angular/Angular_NET5_APP/ProEventos.Domain/Evento.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ProEventos.Domain
{
    //[Table("TB_EVENTOS_DETALHES")] caso eu quisesse mudar o nome da tabela
    public class Evento
    {
        [Key]
        public int EventoId { get; set; }
        public string Local { get; set; }
        public DateTime? DataEvento { get; set; }

        // [NotMapped] caso quisesse criar um atributo que não seja necessario passar para o DB.
        // public int Dias_evento { get; set; } 

        [Required]
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string ImagemURL { get; set; }

        [Required]
        public string Telefone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<RedeSocial> RedesSociais { get; set; }
        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
    }


}
